skymp5-server/cpp/server_guest_lib/AnimationSystem.h

```diff
@@ -28,7 +31,15 @@ class AnimationSystem
     MpActor* actor,
     std::chrono::steady_clock::time_point timePoint =
       std::chrono::steady_clock::now());
+  std::vector<uint32_t> GetWeaponKeywordFormIds(uint32_t baseId) const;
+  std::vector<std::string_view> GetWeaponKeywords(uint32_t baseId) const;
+  float ComputeWeaponStaminaModifier(uint32_t baseId) const;
+  void HandleAttackAnim(MpActor* actor, float defaultModifier = 0.f) const;
```

Comment:
defaultModifier -> defaultMultiplier

Please remove `= 0.f`.

i think it'd better to keep it explicit. or we can make zero damage weapon by mistake.

Reply:
As it is said in the doc attached to the asana task associated with this PR, we want to keep old behavior for all other weapons that do not contain specific keywords.

Reply:
Oops, it is actually said in task description.